<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QWidget" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="username_editBalance_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newBalance_lineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>New Balance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editBalnce_pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_addAccount_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_addAccount_lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="address_addAccount_lineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="age_addAccount_lineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="suspended_addAccount_checkBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Suspend</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Activate_addAccount_checkBox_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Activate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addAccount_pushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="suspend_lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="suspend_pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Suspend</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Activate_lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Activate_pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Activate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
